network-streaming-performance-k8s/README.md

```diff
@@ -0,0 +1,89 @@
+# Network Streaming Performance Workflow for Kubernetes
+
+***NOTE: This is a work-in-progress workflow.***
+
+## Workflow Description
+
+This example workflow runs a network streaming bi-directional load using the
+[uperf](https://github.com/uperf/uperf) benchmark utility. The workload will
```

Comment:
Consider using `workflow` instead of `workload` here.

Reply:
"Workload" is intentional here. The "workflow" runs plugins both locally and on the k8s cluster, so "workload" is used to differentiate the plugins running an actual load, which will be scheduled on the k8s cluster.

Reply:
I still find it confusing. But perhaps by working in the explanation you gave above into the text we could avoid that.

Reply:
I updated the language a bit, so I'll resolve this one for now.